src/main/java/com/google/api/codegen/transformer/go/GoGapicSurfaceTestTransformer.java

```diff
@@ -125,73 +120,27 @@ private MockCombinedView generateMockServiceView(
         .build();
   }
 
-  private List<ClientTestCaseView> createTestCaseViews(SurfaceTransformerContext context) {
-    ArrayList<ClientTestCaseView> testCaseViews = new ArrayList<>();
+  private List<TestMethodView> createTestCaseViews(SurfaceTransformerContext context) {
+    ArrayList<TestMethodView> testCaseViews = new ArrayList<>();
     SymbolTable testNameTable = new SymbolTable();
     for (Method method : context.getSupportedMethods()) {
-      MethodConfig methodConfig = context.getMethodConfig(method);
+      MethodTransformerContext methodContext = context.asRequestMethodContext(method);
+      ClientMethodType clientMethodType = ClientMethodType.RequestObjectMethod;
+      if (methodContext.getMethodConfig().isPageStreaming()) {
+        clientMethodType = ClientMethodType.PagedRequestObjectMethod;
+      }
       testCaseViews.add(
-          createTestCaseView(
-              context.asRequestMethodContext(method),
+          mockServiceTransformer.createTestMethodView(
+              methodContext,
               testNameTable,
-              Collections.<FieldConfig>emptyList()));
+              initCodeTransformer.createRequestInitCodeContext(
```

Comment:
Oh I saw the description. No as long as we know the reason I don't think it is a big concern.